QUICKSTART-DEPLOY.md

```
@@ -10,13 +10,31 @@
+```
+
+If the dashboard build fails in Docker, run it locally:
+
+```bash
+docker compose up planetscale agent 
+cd apps/dashboard 
+pnpm i
+pnpm dev
+```
+
+4. Wait for all services to be healthy
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)